src/script/components/RichTextEditor/plugins/TypeaheadMenuPlugin.tsx

```diff
@@ -748,13 +728,14 @@ export function TypeaheadMenuPlugin<TOption extends TypeaheadOption>({
     };
   }, [editor, triggerFn, onQueryChange, resolution, closeTypeahead, openTypeahead, menuVisible, setMenuVisible]);
 
-  return resolution === null || editor === null ? null : (
+  return resolution === null || editor === null || options.length === 0 ? null : (
```

Comment:
making sure we don't display the component if there are no options to show the user